logstash-core/src/main/java/org/logstash/plugins/PluginFactoryExt.java

```diff
@@ -56,6 +57,11 @@
 
 public final class PluginFactoryExt {
 
+    @FunctionalInterface
+    public interface PluginSearcher {
```

Comment:
nitpick: maybe name it `PluginLocator` or `PluginResolver#resolve` if it sounds clearer to you ...

Reply:
Thanks @kares I pushed the nitpicks